### PR TITLE
write document api get user infomation

### PR DESCRIPTION
**Pull request by ghost**
## Description
write document api get user infomation
## Reference Requirements
- Task ID: TI-1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Comment by at-dungle** [comment]
LGTM

File: docs/api-references/user.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+| user | int | required | User's id |
+```json
+{
+    "meta": {
+        "status": "successfuly",
+        "code": 200
+    },
+    "data": {
+        "id": 8,
+            "name": "Prof. Nestor Reinger III",
```
Comment (at-tuanphan) [inline docs/api-references/user.md]: sai code conventions rồi em

**Review by ghost — COMMENTED** [review]
please check my comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**